src/components/functions/supported.metadata.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div id = "GetClientMetadata">
```

Comment:
Remove all of these unused and useless ids:
```suggestion
  <div>
```

Reply:
Yep!